### PR TITLE
Release Note for TELCODOCS-782 - workload partitioning

### PR DESCRIPTION
**Pull request by slovern**
Version(s): 4.13

Issue: https://issues.redhat.com/browse/TELCODOCS-782

Link to docs preview: https://58139--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-ran-workload-partitioning-three-node-standard-node

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -411,6 +411,13 @@ This update adds the following features:
 
 For more information, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.adoc#cnf-about-numa-aware-scheduling_numa-aware[Scheduling NUMA-aware workloads].
 
+[id="ocp-4-13-workload-partitioning"]
+==== Workload partitioning
+
+You can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
```
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:417]:
```suggestion
Previously, workload partitioning was supported for {sno} clusters only.
Now, you can also configure workload partitioning for three-node compact clusters and standard clusters.
Use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
```

**Review by aireilly — CHANGES_REQUESTED** [review]
Nice work. 

General comment - this release note seems to miss the fact that workload partitioning has been extended to cover 3NC and standard clusters. Previously, only SNO clusters could have workload partitioning.  

My suggested updates incorporate this change.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -411,6 +411,13 @@ This update adds the following features:
 
 For more information, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.adoc#cnf-about-numa-aware-scheduling_numa-aware[Scheduling NUMA-aware workloads].
 
+[id="ocp-4-13-workload-partitioning"]
+==== Workload partitioning
```
[the remark this answers is not in the document]
Comment (slovern) [reply release_notes/ocp-4-13-release-notes.adoc]: Thanks - done

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -495,6 +495,15 @@ This update adds the following features:
 
 For more information, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.adoc#cnf-about-numa-aware-scheduling_numa-aware[Scheduling NUMA-aware workloads].
 
+[id="ocp-4-13-ran-workload-partitioning-three-node-standard-node"]
+==== Support for workload partitioning for three-node clusters and standard clusters 
+
+Previously, workload partitioning was supported for {sno} clusters only.
```
Comment (GroceryBoyJr) [inline release_notes/ocp-4-13-release-notes.adoc:501]:
```suggestion
Before this update, workload partitioning was supported for {sno} clusters only.
```

**Review by GroceryBoyJr — COMMENTED** [review]
@slovern only one tiny nit, otherwise LGTM. Happy to see your link works beautifully! :+1:

File: release_notes/ocp-4-13-release-notes.adoc (hunk 3 above)
Comment (slovern) [reply release_notes/ocp-4-13-release-notes.adoc]: Thanks for the review @GroceryBoyJr

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**